tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -598,6 +599,40 @@ def test_performance_issue_ids_filter(self):
         assert response.status_code == 200, response.content
         assert response.data["data"][0]["count()"] == 1
 
+    def test_generic_issue_ids_filter(self):
+        user_data = {
+            "id": self.user.id,
+            "username": "user",
+            "email": "hellboy@meow.com",
```

Comment:
nit: Love hellboy@meow 🐱, but is there any concern with meow.com being a real domain?

Reply:
Hah I hadn't thought about that, I'll change it.